app/models/zendesk/zendesk_tickets.rb

```diff
@@ -1,14 +1,12 @@
 module Zendesk
   class ZendeskTickets
     def raise_ticket(ticket_to_raise)
-      ZendeskTicketWorker.perform_async({
-        :subject => ticket_to_raise.subject,
-        :priority => ticket_to_raise.priority,
-        :requester => {"locale_id" => 1, "email" => ticket_to_raise.email, "name" => ticket_to_raise.name},
-        :collaborators => ticket_to_raise.collaborator_emails,
-        :tags => ticket_to_raise.tags,
-        :comment => { "body" => ticket_to_raise.comment }
-      })
+      ZendeskTicketWorker.perform_async(subject: ticket_to_raise.subject,
+        priority: ticket_to_raise.priority,
+        requester: { "locale_id" => 1, "email" => ticket_to_raise.email, "name" => ticket_to_raise.name },
+        collaborators: ticket_to_raise.collaborator_emails,
+        tags: ticket_to_raise.tags,
+        comment: { "body" => ticket_to_raise.comment })
```

Comment:
This looks pretty weird.  Would it be happy with:

```ruby
ZendeskTicketWorker.perform_async(
  subject: ticket_to_raise.subject,
  priority: ticket_to_raise.priority,
  requester: { "locale_id" => 1, "email" => ticket_to_raise.email, "name" => ticket_to_raise.name },
  collaborators: ticket_to_raise.collaborator_emails,
  tags: ticket_to_raise.tags,
  comment: { "body" => ticket_to_raise.comment })
)
```

instead? (E.g. keep the new hash syntax and drop the redundant `{}` but don't move the first part of the hash onto the same line as the method call.)